data-raw/material_thesaurus.csv

```diff
@@ -1,249 +1,504 @@
 cor,var
+faeces,Dung
+micro organism,Foraminifera
+plant remains,grass cuticle
+NA,nd
+NA,---
+plant remains,(Acacia?) roots
+NA,>125 micron organic material
+shell,Acanthopleura sp.
+plant remains,acorn
+plant remains,acorns
+plant remains,Acorns
+bone,aegilops (saiga)
+NA,AH
+undefined,alkaline insoluble
+shell,Anadara antiquata
+shell,Anadara granosa shell
+shell,Anadara sp. shell
+shell,"Anadara sp., Tapes hiantina"
+shell,Anadara trapezia shell
+bone,animal bone
+bone,Animal Bone
+hair,animal hair
+antler,antler
+plant remains,apple
+aragonite,aragonite
+aragonite,aragonte
 ash,ash
+undefined,Ash and charcoal
+undefined,Ash and shell
 ash,ashy soil
+undefined,Austomylitus rostratus shell
+wood,Axtschaft
+NA,B
+ceramics,Baked clay from hearth
+shell,Barbatia helbingi
+shell,Barbatia sp. Shell
 bark,bark
-bark,Rinde
-bark,riinde
-brain,brain
-brain,brain tissue
+bark,Bark
+plant remains,barley
+NA,bd
+plant remains,bean
+plant remains,beans
+plant remains,been
+undefined,Beeswax
+plant remains,bine
+calcium carbonate,Bird eggshell
 bone,bone
-bone,bones
 bone,Bone
-bone,cremated bones
-bone,human bone
-bone,human
-bone,"collagen, bone"
-bone,collagen
+bone,Bone (apatite fraction)
+bone,bone (Bos taurus)
+bone,Bone (collagen not removed)
+bone,Bone (collagen)
+bone,Bone (extracted organic material)
+bone,Bone (free carbonate)
+bone,Bone (gelatin)
+bone,Bone (insoluble residue)
+bone,Bone (Macropus rufogriseus)
+bone,Bone (non-apatite fraction)
+bone,Bone (non-humic fraction)
+bone,Bone (organic carbon)
+bone,Bone (organic fraction)
+bone,Bone (organics)
+bone,Bone (re-precipitated alkali soluble fraction)
+bone,Bone (residue)
+bone,bone (Sus domestica)
+bone,bone (Sus scrofa)
+bone,Bone (Vombatus ursinus)
+bone,Bone apatite
+bone,Bone apatite/carbonate
+bone,Bone carbonate
 bone,"bone, rib"
+NA,bone?
 bone,bone/apatit
 bone,bone/collagen
-bone,animal bone
-bone,Animal Bone
-bone,bone (Bos taurus)
+bone,bones
 bone,bos
-bone,Ur
-bone,bone (Sus domestica)
-bone,bone (Sus scrofa)
-bone,aegilops (saiga)
-bone,pig
-bone,hueso
-ceramics,pottery
+NA,Br
+shell,Brachidontes rostratus shell
+tissue,brain
+tissue,brain tissue
+food,bread
+NA,Bulk
+NA,bulk
+undefined,Bulk organic fraction
```

Comment:
maybe we just ignore the "bulk" part and add the to the respective categories

Reply:
please elaborate